emerge/languages/javascriptparser.py

```diff
@@ -113,63 +114,71 @@ def _add_imports_to_result(self, result: AbstractResult, analysis):
         filtered_list_no_comments = self.preprocess_file_content_and_generate_token_list_by_mapping(source_string_no_comments, self._token_mappings)
 
         for _, obj, following in self._gen_word_read_ahead(filtered_list_no_comments):
-            if obj == JavaScriptParsingKeyword.IMPORT.value:
-                read_ahead_string = self.create_read_ahead_string(obj, following)
+            if obj != JavaScriptParsingKeyword.IMPORT.value and obj != JavaScriptParsingKeyword.REQUIRE.value:
```

Comment:
At first, I wanted to extract a method, but I realized I was inside a loop. I made it work by early-returning for cases that won't match.

Then, the difference is the `expression_to_match` that would be different regarding the import keyword used.

Reply:
Nice way of dealing with both import cases 👍